modules/serverless-list-subs-kn.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+//  * serverless/event_workflows/serverless-subs.adoc
+
+[id="serverless-list-subs-kn_{context}"]
+= Listing subscriptions using the Knative CLI
+
+You can list existing subscriptions on your cluster by using the `kn` CLI.
+
+.Prerequisites
+
+* You have installed the `kn` CLI.
+
+.Procedure
+
+* List subscriptions on your cluster:
++
+[source,terminal]
+----
+$ kn subscription list
+----
++
+.Example output
+[source,terminal]
+----
+NAME            CHANNEL             SUBSCRIBER           REPLY   DEAD LETTER SINK   READY   REASON
+mysubscription   Channel:mychannel   ksvc:event-display                              True
```

Comment:
```suggestion
mysubscription  Channel:mychannel   ksvc:event-display                              True
```

Reply:
actually, to be correct wrt. what the "kn subscription list" actually shows, we'd add a space on the previous line, not remove the one here... (but I don't think it really matters, as long as it look OK..)

Reply:
("kn subscription list" will leave exactly 3 space characters between "mysubscription   Channel:mychannel" and 13 space characters between "NAME             CHANNEL"

Reply:
@maschmid better? :)